src/style.css

```diff
@@ -1,3 +1,99 @@
+/* zeroing */
+
+img {
+  vertical-align: top;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul,
+li {
+  list-style: none;
+}
+
+html,
 body {
+  font-family: Roboto, sans-serif;
+  color: #000;
+  font-size: 12px;
   margin: 0;
+  line-height: 1;
+  box-sizing: border-box;
+}
+
+/* -------------------------- wrapper -------------------------- */
```

Comment:
Remove comments. Fix it everywhere. Try it write obvious code that explain all we need without comments)